src/main/java/theChillys/chillys_radio/station/dto/StationResponseDto.java

```diff
@@ -11,7 +11,7 @@
 @Setter
 public class StationResponseDto {
 
-    private Long id;
+    private long id;
```

Comment:
WHY??? All variable names were the same as in the source API. Also types is better to use Linked for better working with collections.

Reply:
Didn't change this class at all, I guess I haven't updated my project in a long time